servers/visual_server.cpp

```diff
@@ -364,6 +368,10 @@ Vector2 VisualServer::tangent_to_oct(const Vector3 v, const float sign, const bo
 // Convert Octohedron-mapped normalized vector back to Cartesian
 // Assumes normalized format (elements of v within range [-1, 1])
 Vector3 VisualServer::oct_to_norm(const Vector2 v) {
+	if (Math::is_equal_approx(v.length(), 0)) {
+		return Vector3(0, 0, 0);
+	}
+
```

Comment:
This doesn't have a divide and Vector2(0, 0) should be valid input. Perhaps this could just have a warning when outside the -1,1 range?

Reply:
its mostly just so that if it's zero its returning before running the rest of the code since the rest is pointless with zero